lib/nrf_modem_lib/Kconfig.modemlib

```diff
@@ -189,6 +189,20 @@ config NRF_MODEM_LIB_MEM_DIAG_DUMP_PERIOD_MS
 
 endif
 
+menu "Interrupt priorities"
+
+DT_IPC := $(dt_nodelabel_path,ipc)
+
+config NRF_MODEM_IPC_IRQ_PRIO_DTS_OVERRIDE
```

Comment:
@eivindj-nordic do you think these should have the `_LIB_` prefix?
They don't belong to libmodem itself.

Reply:
Yes, they should!